### PR TITLE
geojson support for markerData

### PR DESCRIPTION
**Pull request by nmccready**

### CODE REVIEW

**Comment by nmccready** [comment]
@zacronos @tombatossals

File: src/services/leafletIterators.js [hunk 1]
```diff
@@ -1,44 +1,166 @@
...
-            }
-        }
-        if(!$lhelp.isFunction(cb)){
-            cbName = $lhelp.defaultTo(cb,'cb');
-            $log.error(errorHeader + cbName + ' is not a function');
-            return true;
-        }
-        return false;
+  var lHlp = leafletHelpers,
+  errorHeader = leafletHelpers.errorHeader + 'leafletIterators: ';
```
Comment (nmccready) [inline src/services/leafletIterators.js]: copying over underscore functionality from underscore

File: test/unit/bootstrap.coffee [hunk 2]
```diff
@@ -0,0 +1,12 @@
+window.ngLeafLetTestGlobals = {}
+
+beforeEach ->
+
```
Comment (nmccready) [inline test/unit/bootstrap.coffee]: DRY

File: test/unit/bootstrap.coffee [hunk 3]
```diff
@@ -0,0 +1,12 @@
+window.ngLeafLetTestGlobals = {}
```
Comment (nmccready) [inline test/unit/bootstrap.coffee]: this should probably go away and have only one injector in the bootstrap

File: test/unit/markersDirectiveSpec.js [hunk 4]
```diff
@@ -1,699 +0,0 @@
-'use strict';
-
```
Comment (nmccready) [inline test/unit/markersDirectiveSpec.js]: moved to *.coffee

File: test/unit/services/leafletGeoHelpersSpec.coffee [hunk 5]
```diff
@@ -0,0 +1,59 @@
+describe 'leafletGeoJsonHelpers', ->
```
Comment (nmccready) [inline test/unit/services/leafletGeoHelpersSpec.coffee]: specs to show validation of a Point (lat: lng) . Also verify the creation / usage of geojson Point and Array [lng,lat].

File: examples/marker-example-geojson-data.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    <script>
+        var app = angular.module("demoapp", ["leaflet-directive"]);
+        app.controller('DemoController', [ '$scope', function($scope) {
+            angular.extend($scope, {
+                london: {
+                    lat: 51.505,
+                    lng: -0.09,
+                    zoom: 8
+                },
+                markers: {
```
Comment (nmccready) [inline examples/marker-example-geojson-data.html]: Example to show the geojson data is rendered.

**Comment by zacronos** [comment]
Hmm, I think this PR needs more jQuery -- please fix before merge.

**Comment by zacronos** [comment]
Happy April Fools!  (LGTM)

**Comment by nmccready** [comment]
lol

**Pull request merged**